compiler/package-lock.json

```diff
@@ -33,6 +33,10 @@
         "node": ">=14"
       }
     },
+    "../compiler-rs/compiler-wasm-lib/pkg": {
```

Comment:
Hm, I'm not sure we should be touching this file

Reply:
Yes, it would be nice to revert those unrelated changes (also in package.json).

Reply:
The PR has a mix of manual changes and automated changes from just building and validation ... should i revert all changes that result from the automation ? (if so, should these be .gitignored instead)

Reply:
Please keep the changes in specification/ (manual) and output/ (generated) only. Thank you!

Reply:
done in https://github.com/elastic/elasticsearch-specification/pull/3125/commits/ca540518dd3a8e9c887135179aca043f23252739